app/assets/stylesheets/pertaxMain.scss

```diff
@@ -156,14 +156,29 @@ input[type="text"].autocomplete__input {
 .covid-banner__title {
     font-weight: 700;
     float: left;
+
+    display: inline-block;
+    display: -ms-flexbox;
+    display: flex;
+    -ms-flex-align: center;
+    align-items: center;
+    box-sizing: border-box;
+    padding-left: 50px;
+    min-height: 45px;
+    background-image: url(https://assets.publishing.service.gov.uk/static/covid-arrow-af5ae4c3e1f65811d4e472b4a5dace2950508ad3ef956b54e0cdfc80aef96419.png);
```

Comment:
should we be hosting the image or is this standard asset stuff?

Reply:
thats where gov.uk is hosting the image